repos/spack_repo/builtin/packages/hepmc3/package.py

```diff
@@ -46,9 +46,7 @@ class Hepmc3(CMakePackage):
         description="Install interfaces for some Monte-Carlo Event Gens",
     )
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
+    depends_on("cxx", type="build")
```

Comment:
It looks like `@3.1:3.2` require C implicitly for configure.

https://gitlab.cern.ch/hepmc/HepMC3/-/blob/3.2.7/CMakeLists.txt?ref_type=heads#L3

Reply:
```suggestion
    depends_on("c", type="build", when="@3.1:3.2")
    depends_on("cxx", type="build")
```

Fortran is only used for `examples/Pythia6Example` (which is not supported in spack; no `+examples` variant).